docs/source/optimizations/eeroq/eeroq_qss.ipynb

```diff
@@ -0,0 +1,518 @@
+      ]
+     },
+     "execution_count": 19,
+     "metadata": {},
+     "output_type": "execute_result"
+    }
+   ],
+   "source": [
+    "remove_idle_wires(cc).draw()"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "cd09e5d5-60f2-4361-b879-f2479df82e70",
+   "metadata": {},
+   "source": [
+    "# Circuit Simulation"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 20,
+   "id": "81f64cd2-7735-4a95-aada-1efa1a0da946",
+   "metadata": {},
+   "outputs": [
+    {
+     "data": {
+      "text/html": [
+       "<pre style=\"word-wrap: normal;white-space: pre;background: #fff0;line-height: 1.1;font-family: &quot;Courier New&quot;,Courier,monospace\">     ┌────────────┐     ┌────────────┐┌─┐\n",
+       "q_0: ┤ Rx(2.2136) ├──■──┤ Rx(4.5827) ├┤M├\n",
+       "     └─────┬──────┘┌─┴─┐└────┬─┬─────┘└╥┘\n",
+       "q_1: ──────■───────┤ S ├─────┤M├───────╫─\n",
+       "                   └───┘     └╥┘       ║ \n",
+       "c: 2/═════════════════════════╩════════╩═\n",
+       "                              1        0 </pre>"
+      ],
+      "text/plain": [
+       "     ┌────────────┐     ┌────────────┐┌─┐\n",
+       "q_0: ┤ Rx(2.2136) ├──■──┤ Rx(4.5827) ├┤M├\n",
+       "     └─────┬──────┘┌─┴─┐└────┬─┬─────┘└╥┘\n",
+       "q_1: ──────■───────┤ S ├─────┤M├───────╫─\n",
+       "                   └───┘     └╥┘       ║ \n",
+       "c: 2/═════════════════════════╩════════╩═\n",
+       "                              1        0 "
+      ]
+     },
+     "execution_count": 20,
+     "metadata": {},
+     "output_type": "execute_result"
+    }
+   ],
+   "source": [
+    "from qiskit.circuit.random import random_circuit\n",
+    "\n",
+    "qc = random_circuit(2, 3, measure=True)\n",
+    "qc.draw()"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "b45eec80",
+   "metadata": {},
+   "source": [
+    "We can perform noisy simulation by setting the \"method='noise-sim'\" and specifying an error rate."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 21,
+   "id": "9f4e257f-8d61-4578-a038-3713a5678fbf",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "job = backend.run(qc, shots=100, method=\"noise-sim\", error_rate=0.01)"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 22,
+   "id": "cf99f946-020d-4108-8aed-a0c3fdb65559",
+   "metadata": {},
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "{'01': 52, '02': 2, '00': 32, '22': 3, '21': 1, '20': 1}\n"
+     ]
+    }
+   ],
+   "source": [
+    "print(job.result().get_counts())"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "e780139f-2969-4e8c-b006-e84cb0860c77",
+   "metadata": {},
+   "source": [
+    "A measurement value of 2 refers to an out-of-codespace error."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "1731f1cb-6ac2-45c5-afa5-b9d25efbd767",
+   "metadata": {},
+   "outputs": [],
+   "source": []
+  }
```

Comment:
nit: it would be nice to remove the stray empty cell at the end of the notebooks (as most of the other notebooks don't have them)